tools/noise/gen-profile

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This really shouldn't be in this commit, but it fixes color output from the script.